### PR TITLE
New Model 'Book'

### PR DESCRIPTION
**Pull request by mcolombo87**
### Summary
New Model "Book" was created and validated it through rails console.
### ScreenShots
image
image
### Trello
https://trello.com/c/NpV8mWPc/19-crear-modelo-de-book
### Known Issues/Notes
I changed string type to integer type for "Year" field, that is more typical.

### CODE REVIEW

File: db/migrate/20181129173239_create_books.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CreateBooks < ActiveRecord::Migration[5.2]
+  def change
+    create_table :books do |t|
+      t.string :genre
```
Comment (facucompe) [inline db/migrate/20181129173239_create_books.rb]: Add, null constraints. It's good to have them despite the rails validations

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    t.string "author"
+    t.string "image"
+    t.string "title"
+    t.string "editor"
+    t.integer "year"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "users", force: :cascade do |t|
```
Comment (facucompe) [inline db/schema.rb]: Careful with this. Users are not part of this PR

**Review by facucompe — CHANGES_REQUESTED** [review]

File: db/migrate/20181129173239_create_books.rb (hunk 1 above)
Comment (mcolombo87) [reply db/migrate/20181129173239_create_books.rb]: done

File: db/schema.rb (hunk 2 above)
Comment (mcolombo87) [reply db/schema.rb]: done

**Review by facucompe — APPROVED** [review]
